test/net/sourceforge/kolmafia/session/EquipmentManagerTest.java

```diff
@@ -180,4 +188,33 @@ public void canParseStatus() {
       assertItemUnequip(Slot.FOLDER5);
     }
   }
+
+  @Nested
+  class professor {
+    @ParameterizedTest
+    @CsvSource({
+      "mafia thumb ring, false",
+      "Treads of Loathing, false",
+      "panhandle panhandling hat, false",
+      "batskin belt, true",
+      "mafia wedding ring, true"
+    })
+    public void itShouldEquipWhatWasRequestedForProf(String item, String equippable) {
+      AdventureResult itemAR = ItemPool.get(item);
+      boolean canBeEquipped = Boolean.parseBoolean(equippable);
```

Comment:
should be able to be `boolean equippable`, saving you the need to parse it.

Reply:
Done.  Thank you.